src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -197,6 +248,12 @@
       <OutputFolderForTestDependencies>$(BinDir)/TestDependencies</OutputFolderForTestDependencies>
     </PropertyGroup>
 
+    <!-- For UAP we need to copy the generated resources.pri containing the framework resources to the test folder -->
+    <Copy Condition="'$(BuildingUAPVertical)' == 'true' AND '$(TestProjectNeedsModifiedPriFile)' != 'true'"
+          SourceFiles="$(CommonPriFile)"
```

Comment:
We generally want to avoid extra custom copies and get the full set of files to copy into the RunnerFolderContents item which gets used for our script generation.